### DISCUSSION TITLE
Better default Material You Plasma color scheme

### DISCUSSION THREAD

**Discussion by luisbocanegra** [luisbocanegra/kde-material-you-colors#194] [Ideas]
**Is your feature request related to a problem? Please describe.**
As of version [v1.5.1](https://github.com/luisbocanegra/kde-material-you-colors/releases/tag/v1.5.1) the default Plasma color scheme uses some colors that I decided after some time of experimenting and trying to match [Material 3 (You) guidelines](https://m3.material.io/) but not thinking much about it, also because those guidelines were on a very early state.

**Describe the solution you'd like**
I'm making this Issue so people can help proposing/making changes to the default color scheme what we need is:
- [ ] A relationship between every [Plasma color scheme element](https://invent.kde.org/plasma/breeze/-/tree/master/colors), you can read the  [KDE Human Interface Guidelines](https://develop.kde.org/hig/) > [Style](https://develop.kde.org/hig/style/) > [Color](https://develop.kde.org/hig/style/color/) and [KDE Human Interface Guidelines](https://develop.kde.org/hig/) > [Components](https://develop.kde.org/hig/components/) to M3 color [roles or tokens](https://m3.material.io/styles/color/the-color-system/key-colors-tones) that better match components M3 [components](https://m3.material.io/components)

- ~~[x] An illustrated figma like, or at least images detailing what every Plasma color scheme value corresponds to which Plasma/QWidget component, I don't know if such a thing exists yet, probably does.~~ overkill

Some probably useful links 
https://docs.kde.org/trunk5/en/plasma-workspace/kcontrol/colors/index.html
https://develop.kde.org/docs/extend/plasma/theme/theme-details/#colors

https://github.com/material-foundation/material-theme-builder
https://github.com/material-components/material-web#readme
https://community.kde.org/Get_Involved/design#Current_Projects
https://www.figma.com/@materialdesign

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by luisbocanegra** [comment]
Which looks better? First is current
![b](https://github.com/luisbocanegra/kde-material-you-colors/assets/15076387/9b6a7998-e0ce-4724-8110-653d78c57bf7)

![c](https://github.com/luisbocanegra/kde-material-you-colors/assets/15076387/0a848faf-db03-44cf-a6aa-6cd7458aef8f)

**Reply by ShatteredSkies** [reply]
i feel like the third one looks best, darker background contrasts the accent color better.
others has light tint accent color which i don't particularly like. (also the text is white on a light background)

also, since i use light mode i wonder how it looks like?

**Reply by luisbocanegra** [reply]
here's the light mode of the third one
![image](https://github.com/luisbocanegra/kde-material-you-colors/assets/15076387/19b4c3a1-922f-439f-b7f5-3e4f8ec8d835)

**Reply by ShatteredSkies** [reply]
looks good enough.
the problem now is the dark text on dark background.. is that a KDE issue?

**Reply by luisbocanegra** [reply]
For the selection (accent colored) areas? That's a KDE issue changing the text color when they are inactive even when `ChangeSelectionColor=false` is set in the color scheme file, I was trying to change it but seems hardcoded, reverting that restores the current bahavior:
![d](https://github.com/luisbocanegra/kde-material-you-colors/assets/15076387/55f6f43a-bdf4-4a40-861b-4716d46bfb8c)

**Reply by luisbocanegra** [reply]
Should inactive windows have some sort of color change or effect applied (e.g. darkening/desaturation)?

**Reply by ShatteredSkies** [reply]
i personally don't like inactive color, but i think it would be nice to have it as an option.
